### DISCUSSION TITLE
[FIXED] Client capability :workspaceEdit :documentChanges is :json-false, why?

### DISCUSSION THREAD

**Discussion by nemethf** [joaotavora/eglot#853] [General]
It seems Eglot does support documentChanges with versions and everything.  Why is this capability false then?

**Comment by joaotavora** [comment]
What does the Git history say? Maybe it's just a bug...

**Reply by nemethf** [reply]
I lost my way in the numerous refactorings as I'm no git archaeologist but I believe the capability appeared first in
4c0bfc3, May 14 2018 and the feature earlier in 6e76b08, May 8 2018 or in 3a6c637, May 8 2018.  So it's not like someone just failed to change :json-false to t, it was deliberately set to false.  But maybe that was a thinko.  

If it is OK to generate an error when the version of the document is newer than the version in the requested change, then I think :docuementChanges should be changed to t.

**Reply by nemethf** [reply]
Commit 9389d2e4a097 fixed this.
